### PR TITLE
Enhancement / expression functions: distance

### PR DESCRIPTION
**Pull request by SteRiccio**

### CODE REVIEW

File: src/srs/index.ts [hunk 1]
```diff
@@ -1,22 +1,3 @@
-export interface SRS {
-  code: string
-  name: string
-  wkt: string
-}
+export { DEFAULT_SRS, SRS } from './srs'
```
Comment (minotogna) [inline src/srs/index.ts:1]: SRS must be exported with type. can u check if there's a es lint rule that prevents exporting types without type?

e.g. https://github.com/intuit/eslint-plugin-no-explicit-type-exports

**Review by minotogna — CHANGES_REQUESTED** [review]
we use yarn, not npm. remove package.json

**Comment by SteRiccio** [comment]
> we use yarn, not npm. remove package.json

You mean package-lock.json, right?

File: src/srs/index.ts (hunk 1 above)
Comment (SteRiccio) [reply src/srs/index.ts]: fixed

**Review by minotogna — APPROVED** [review]

**Review by ramirobg94 — APPROVED** [review]

**Review by sorja — APPROVED** [review]

**Pull request merged**